src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -152,11 +175,11 @@ def generate_config(self, daemon_spec: CephadmDaemonDeploySpec) -> Tuple[Dict[st
             addr = self.mgr.inventory.get_addr(dd.hostname)
             dashboard_urls.append(build_url(scheme=proto, host=addr, port=port))
 
-        context = {
-            'dashboard_urls': dashboard_urls,
-            'default_webhook_urls': default_webhook_urls
-        }
-        yml = self.mgr.template.render('services/alertmanager/alertmanager.yml.j2', context)
+        context = AlertmanagerYamlContext(
+            dashboard_urls=dashboard_urls,
```

Comment:
What's the reason for not keeping default_webhook_urls here? I know it got moved from part of user_data to it's own thing but why not still include it in the context?

Reply:
my resoning was: we're not doing anything with it. we're just passing it to jinja2. That's why I removed it completely from the python code base